sqlglot/dialects/redshift.py

```diff
@@ -31,6 +31,7 @@ class Redshift(Postgres):
     RESOLVES_IDENTIFIERS_AS_UPPERCASE = None
 
     SUPPORTS_USER_DEFINED_TYPES = False
+    INDEX_OFFSET = 0
```

Comment:
FYI I tested this in Redshift directly and it can also be seen in their [docs](https://docs.aws.amazon.com/redshift/latest/dg/query-super.html) too:

<img width="767" alt="Screenshot 2023-10-04 at 6 58 37 PM" src="https://github.com/tobymao/sqlglot/assets/46752250/c4362fe1-081b-4d05-b80b-b7b15cd34057">